### DISCUSSION TITLE
area给y轴设置domainMin时显示样式问题

### DISCUSSION THREAD

**Discussion by lumdzeehol** [antvis/G2#5956] [Q&A] [answered]
### 问题描述

面积图设置 `chart.area().scale('y', { type: 'linear', domain: [20000, 30000] });` 后， 区域显示会下探到x轴以下

### 重现链接

https://codesandbox.io/p/sandbox/interesting-brook-fnxwdj

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

### 屏幕截图或视频（可选）
![截屏2023-12-20 02 00 30](https://github.com/antvis/G2/assets/14961982/f6d34251-610f-4a4d-a34c-c23ff7a0d405)

_No response_

### 补充说明（可选）

V4中设置了min 后，小于min的区域不会显示出来， V5 中的表现不一致了

**Comment by pearmini** [comment, marked as answer]
如果希望不展示的话，可以设置 `const chart = new Chart({ clip: true})`

**Reply by lumdzeehol** [reply]
感谢
